src/mame/machine/amstrad.cpp

```diff
@@ -26,7 +26,7 @@ The Z80 must also access the same ram.
 To maintain the screen display, the Z80 is halted on each memory access.
 
 The result is that timing for opcodes, appears to fall into a nice pattern,
-where the time for each opcode can be measured in NOP cycles. NOP cycles is
+where the time for each opcode can be measured in NOP cycles. NOP cycle is
```

Comment:
Plural is correct here – a NOP takes multiple cycles to complete.

Reply:
Agreed